docs/framework/unmanaged-api/profiling/toc.yml

```diff
@@ -444,6 +444,39 @@
         href: icorprofilerinfo7-getinmemorysymbolslength-method.md
       - name: ReadInMemorySymbols
         href: icorprofilerinfo7-readinmemorysymbols.md
+    - name: ICorProfilerInfo8 Interface
+      href: icorprofilerinfo8-interface.md
+      items:
+      - name: IsFunctionDynamic Method
```

Comment:
The three members are in reverse alphabetical order instead of alphabetical order.